### PR TITLE
Added 2016 Hamilton Ohio Precincts

### PR DESCRIPTION
**Pull request by hodgesmr**
Relevant to #7:

I used [pyesridump](https://github.com/openaddresses/pyesridump) to pull the Hamilton County, Ohio precinct geojson from [Cincinnati Area Geographical Information Services](https://cagisonline.hamilton-co.org/arcgis/rest/services/Countywide_Layers/Voting_Precincts/MapServer/1). I then loaded that into QGIS to generate the shapefiles.

This is my first PR to this repo. I'm just kindof guessing on how to structure the submitted files. Please let me know if I need to change the zip of folder structure!

### CODE REVIEW

**Comment by hodgesmr** [comment]
@nvkelso Thanks for pointing that out. Let me know if I did anything else dumb here (not quite sure what to put in the lat_n and lon_n values).

**Comment by nvkelso** [comment]
Same changes need to be made over at https://github.com/nvkelso/election-geodata/pull/138.

File: Makefile [hunk 1]
```diff
@@ -723,6 +724,10 @@ out/39-ohio/state.gpkg: data/39-ohio/statewide/2010/tl_2012_39_vtd10.zip \
 	ogr2ogr -sql "SELECT '2016' AS year, '39' AS state, '021' AS county, CONCAT('39021', CAST(District_N AS character(10))) AS precinct, 'polygon' AS accuracy FROM CHP_PREC" \
 		-s_srs '+proj=lcc +lat_1=38.73333333333333 +lat_2=40.03333333333333 +lat_0=38 +lon_0=-82.5 +x_0=600000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
 		-t_srs EPSG:4326 -nln state -append -f GPKG $@ 'out/39-ohio/source/CHP_PREC.shp'
+	unzip -d out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip
```
Comment (nvkelso) [inline Makefile]: This one tossed me for a loop:

> unzip -d out/39-ohio/source data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

versus:

> unzip -d out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

The output and input both need to be specified else unzip will choke.

Comment (hodgesmr) [reply Makefile]: Aha, good catch. Messed up a paste on that. Fixed.

File: Makefile [hunk 2]
```diff
@@ -724,7 +724,7 @@ out/39-ohio/state.gpkg: data/39-ohio/statewide/2010/tl_2012_39_vtd10.zip \
 	ogr2ogr -sql "SELECT '2016' AS year, '39' AS state, '021' AS county, CONCAT('39021', CAST(District_N AS character(10))) AS precinct, 'polygon' AS accuracy FROM CHP_PREC" \
 		-s_srs '+proj=lcc +lat_1=38.73333333333333 +lat_2=40.03333333333333 +lat_0=38 +lon_0=-82.5 +x_0=600000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
 		-t_srs EPSG:4326 -nln state -append -f GPKG $@ 'out/39-ohio/source/CHP_PREC.shp'
-	unzip -d out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip
+	unzip -d out/39-ohio/source data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip
```
Comment (nvkelso) [inline Makefile]: Hmm, this is a mixed bag:

> unzip -d out/39-ohio/source/39061-hamilton/ data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

To match with the later command's path:

> 'out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.shp'

**or**

> unzip -d out/39-ohio/source/ data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

But update the later command's path:

> 'out/39-ohio/HamiltonCountyOHPrecincts_2016.shp'

Comment (hodgesmr) [reply Makefile]: It looks like the common patter for the other files is:

`unzip -d out/39-ohio/source/ data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip`

and

`'out/39-ohio/source/HamiltonCountyOHPrecincts_2016.shp'`

**Pull request merged**